### ISSUE TITLE
`pyglet.gl` has no attribute 'GL_MAX_TEXTURE_MAX_ANISOTROPY' in `pyglet==2.0.dev23`

### ISSUE THREAD

**Issue by jmuchovej** [pythonarcade/arcade#1352]
## System Info

```shell
$ python -m arcade      

Arcade 2.6.16
-------------
vendor: Apple
renderer: Apple M1 Pro
version: (4, 1)
python: 3.10.6 | packaged by conda-forge | (main, Aug 22 2022, 20:38:29) [Clang 13.0.1 ]
platform: darwin
```

### Actual behavior:

```shell
$ python tutorial_src.py
Traceback (most recent call last):
  File "/abs/path/to/tutorial_src.py", line 422, in <module>
    main()
  File "/abs/path/to/tutorial_src.py", line 416, in main
    window = MyGame()
  File "/abs/path/to/tutorial_src.py", line 157, in __init__
    super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
  File "/abs/path/to/conda/envs/arcade/lib/python3.10/site-packages/arcade/application.py", line 165, in __init__
    self._ctx: ArcadeContext = ArcadeContext(self, gc_mode=gc_mode)
  File "/abs/path/to/conda/envs/arcade/lib/python3.10/site-packages/arcade/context.py", line 44, in __init__
    super().__init__(window, gc_mode=gc_mode)
  File "/abs/path/to/conda/envs/arcade/lib/python3.10/site-packages/arcade/gl/context.py", line 162, in __init__
    self._limits = Limits(self)
  File "/abs/path/to/conda/envs/arcade/lib/python3.10/site-packages/arcade/gl/context.py", line 1179, in __init__
    self.MAX_TEXTURE_MAX_ANISOTROPY = self.get_float(gl.GL_MAX_TEXTURE_MAX_ANISOTROPY)
AttributeError: module 'pyglet.gl' has no attribute 'GL_MAX_TEXTURE_MAX_ANISOTROPY'
```

### Expected behavior:

The game should launch and run (looks like the attached image).
![Screen Shot 2022-10-09 at 18 52 04](https://user-images.githubusercontent.com/5000729/194783199-b6f264db-f4fb-4af8-8284-a4d057898520.jpg)

### Steps to reproduce/example code:

Copy the source from [Simple Platformer / Step 12][step12] and try to run on the latest install of `arcade`. (Or just run `python -m arcade.examples.platform_tutorial.12_animate_character`.)

[step12]: https://api.arcade.academy/en/latest/examples/platform_tutorial/step_12.html#source-code

### What should be added/changed?

Downgrade to `pyglet==2.0.dev22`. (OR maybe loosen dependency to `pyglet==2.0.*`, since the strict requirement causes `pip` to whine (which might be intimidating for some?).

### What would it help with?

It would return (at least) the `GL_MAX_TEXTURE_MAX_ATISOTROPY` constant (if not others).

**Comment by jmuchovej** [comment]
Seems there was something funky with my virtual environment. A clean install of the environment fixed it. 👀

**Comment by einarf** [comment]
That might happen if you run with software rendering in some setups. We fixed that in arcade 2.7 (current development)
